content/contribute/contributing-and-managing-patterns.adoc

```diff
@@ -0,0 +1,145 @@
+. Can be merged or cherry-picked into your local main branch for use in GitOps workflows (useful for tracking submodule updates, such as common).
+. Serves as a solid basis for Pull Requests upstream, as it will not include local configuration differences or local GitOps commits.
+
+[id="changing-subtrees"]
+== Changing subtrees
+
+Our patterns use the Git subtree feature to promote modularity, enabling multiple patterns to share the same common base. Over time, we move more functionality into the common directory to isolate pattern-specific components as standard usage practices emerge. This approach strengthens the tools in the common directory, adds features, and simplifies the development of new patterns. Typically, we maintain the common subtree during routine updates, and pulling changes from upstream includes any updates from the common directory.
+
+You only need to modify subtrees if you want to test changes in the common area of the pattern repositories or contribute to the common repository alongside one of the patterns. Using the pattern alone does not require changing subtrees.
+
+For most cases involving the use and consumption of the pattern, users do not need to know that the pattern uses a subtree.
+
+[source,terminal]
+----
+$ git clone https://github.com/<your-workspace>/industrial-edge
+----
+
+If you want to modify and track your version of common, fork and clone the common repository separately:
+
+
+[source,terminal]
+----
+$ git clone https://github.com/<your-workspace>/common
+----
+
+Make changes in your fork’s main branch or create a new branch for your modifications.
+
+To track these changes in your fork of the pattern repository (e.g., `industrial-edge`), replace the common subtree with your forked version. To simplify this process use:
+
+[source,terminal]
+----
+$ common/scripts/make_common_subtree.sh <subtree_repo> <subtree_branch> <subtree_remote_name>
+----
+
+This script sets up a new remote in your local working directory with the specified repository. It replaces the common directory with the specified fork and branch, commits the change, but does not push it.
+
+For example:
+
+[source,terminal]
+----
+$ common/scripts/make_common_subtree.sh https://github.com/mhjacks/common.git wip-main common-subtree
+----
+
+This replaces the common directory in the current repository with the wip-main branch from mhjacks's common repository and names the remote `common-subtree`.
+
+To pull changes from mhjacks's wip-main branch into the parent repository:
+
+[source,terminal]
+----
+$ git subtree pull --prefix common common-subtree wip-main
+----
+
+Running the script without arguments defaults to:
+
+[source,terminal]
+----
+$ common/scripts/make_common_subtree.sh https://github.com/validatedpatterns/common.git main common-subtree
+----
+
+These are the default settings for the repository configuration.
+
+[id="subtree-vs-submodule"]
+== Subtree vs. Submodule
```

Comment:
Consider using 'compared  to' rather than 'vs' unless                                 updating existing content that uses the term